### ISSUE TITLE
Hypermedia

### ISSUE THREAD

**Issue by kinlane** [adopta-agency/open-referral-spec#7]
This is the thread to discuss use of hypermedia as part of the HSDS v1.x. Now is the time to have this discussion, and discuss the evolution of the project.

I'm suggesting we look at the following formats, based upon their adoption in the API sector:

JSON API - http://jsonapi.org/
Collection+JSON - http://amundsen.com/media-types/collection/
HAL - Hypertext Application Language  -  http://stateless.co/hal_specification.html
Siren - https://github.com/kevinswiber/siren

I'll be working on a separate entry, outlining the benefits of applying hypermedia design, as well as working examples to help visualize how it will be used.

**Comment by switzersc** [comment]
My comment here is relevant: https://github.com/openreferral/api-specification/issues/6#issuecomment-302951272

tl;dr: Each of the hypermedia types given has pros/cons, and we need to make sure we've thoroughly identified (and documented) our API user stories and use cases to determine the most appropriate one.
